smartctl.go

```diff
@@ -114,19 +115,30 @@ func (smart *SMARTctl) mineDevice() {
 }
 
 func (smart *SMARTctl) mineCapacity() {
-	capacity := smart.json.Get("user_capacity")
+	// The user_capacity exists only when NVMe have single namespace. Otherwise,
+	// for NVMe deivces with multiple namespaces, when device name used without
```

Comment:
> deivces

:point_up_2: :smile:

Reply:
Oh yes, I wrote this with one hand, and with the other I moved my finger in the air telling a colleague, in agony from what I spent the last 3 hours of my life on another debug 🫠